src/hooks/screen/useSignedChatInfoHook.ts

```diff
@@ -2,37 +2,38 @@ import * as React from 'react';
 import {useNavigation} from '@react-navigation/core';
 
 import ChannelList from '../../database/schema/ChannelListSchema';
-import UseAnonymousChatInfoScreenHook from '../../../types/hooks/screens/useAnonymousChatInfoScreenHook.types';
+import UseSignedChatInfoScreenHook from '../../../types/hooks/screens/useSignedChatInfoScreenHook.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `UseSignedChatInfoScreenHook` is new. Ensure that the type definition file `useSignedChatInfoScreenHook.types` exists and is correctly defined.



<!-- This is an auto-generated comment by OSS CodeRabbit -->